modules/programs/iamb.nix

```
@@ -43,11 +45,23 @@ in
 
   };
 
-  config = lib.mkIf cfg.enable {
-    home.packages = lib.mkIf (cfg.package != null) [ cfg.package ];
+  config =
+    let
+      configFile = tomlFormat.generate "iamb-config" cfg.settings;
```

```suggestion
      configFile = tomlFormat.generate "iamb-config" cfg.settings;
      configDir =
        if pkgs.stdenv.isDarwin then
          "Library/Application Support"
        else
          config.xdg.configHome;
```

Could do this, like we do in other modules, and then simplify the config block with one file generation condition using `home.file`